src/python/pants/bin/local_pants_runner.py

```diff
@@ -102,18 +102,11 @@ def create(
         run_tracker = RunTracker(options_bootstrapper.args, options)
         native_engine.maybe_set_panic_handler()
 
-        # Option values are usually computed lazily on demand, but command line options are
-        # eagerly computed for validation.
         with options_initializer.handle_unknown_flags(options_bootstrapper, env, raise_=True):
             # Verify CLI flags.
             if not build_config.allow_unknown_options:
                 options.verify_args()
 
-            for scope, values in options.scope_to_flags.items():
```

Comment:
This should have been removed when we added the new verify_args() logic (via the Rust-side unused arg tracking) above. It was the only remaining use of the `scope_to_flags` field on `Options`.